src/KeyFrame.cpp

```diff
@@ -31,28 +31,32 @@
 #include "KeyFrame.h"
 #include "Exceptions.h"
 
-#include <cassert>         // For assert()
-#include <iostream>        // For std::cout
-#include <iomanip>         // For std::setprecision
 #include <algorithm>
 #include <functional>
 #include <utility>
+#include <cassert>         // For assert()
+#include <iostream>        // For std::cout
+#include <iomanip>         // For std::setprecision
 
 using namespace std;
 using namespace openshot;
 
-namespace {
+namespace openshot{
```

Comment:
The reason these functions were placed in an anonymous namespace is that they're not part of the Keyframe API, they're internal utility functions only visible to functions in `KeyFrame.cpp`.

(See https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces)

They shouldn't be moved to `openshot::` or declared in `KeyFrame.h` unless they need to be exposed beyond this file. (Do they?)

Reply:
I am using the same functions at the TrackedObjectBBox class. To not duplicated code I just included them in the openshot namespace. But if you prefer I can change that